ansible/roles/purdue-rebuilderd/templates/rebuilderd.conf

```diff
@@ -0,0 +1,40 @@
+## Configuration for http daemon
+#[http]
+## The address to bind to. This is 127.0.0.1:8484 by default.
+#bind_addr = "0.0.0.0:8484"
+## If you use a reverse proxy, use this header instead of the actual connecting ip.
+## Make sure the reverse proxy has filters in place to prevent spoofing issues.
+#real_ip_header = "X-Real-IP"
+## Set a default endpoint for rebuildctl. This is especially useful for the sync timer.
+#endpoint = "http://127.0.0.1:8484"
+
+## A random cookie for administration is generated at startup and written to /var/lib/rebuilderd/auth-cookie
+## You can set this to a fixed value here. Use `pwgen -1s 32` to generate one.
+## rebuildctl is searching for this cookie in ~/.config/rebuilderd.conf, /etc/rebuilderd.conf and
+## /var/lib/rebuilderd/auth-cookie in that order.
+#[auth]
+#cookie = "INSECURE"
```

Comment:
I think other than this being templatable this PR looks good right?

Reply:
That'd work. There's currently no vault yet, so if I'd go ahead and create one everybody would need to set it up as well or they won't be able to deploy anymore.

Reply:
yeah, I think that's ok. I think we set a passphrase for the vault already right?

Reply:
Yes, I'm already using it and documented how to configure it in the readme. :)